frontend/src/employee-frontend/components/Header.tsx

```diff
@@ -93,21 +93,35 @@ function HeaderContainer({
   )
 }
 
+const NavLinkWrapper = styled.span`
+  flex: 0 0 auto;
+  display: flex;
+  align-items: center;
+  border-bottom: 4px solid transparent;
+  margin: 6px 24px;
+  padding: 10px 0;
```

Comment:
eVaka logo gets rekt on resolutions lower than 1408px with the added whitespace.

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/1176169/163326814-22a24461-6025-45ce-aec1-4327d2bff2c9.png">

Reply:
Adjusted margins and gaps so things fit a bit better